### PR TITLE
TT-1143 Add styling to saml form

### PR DESCRIPTION
**Pull request by RachaelBooth**
This adds some basic hardcoded styling to passport verify's saml form, so that it doesn't look completely terrible.

### CODE REVIEW

**Comment by andy-paine** [comment]
screen shot 2017-09-25 at 13 38 24

File: lib/saml-form.ts [hunk 1]
```diff
@@ -6,17 +6,31 @@ import * as escape from 'escape-html'
...
+      .passport-verify-saml-form {
+        font-family: Arial, sans-serif;
+      }
+      .passport-verify-button {
+        background-color: #00692f;
+        color: #fff;
+        padding: 10px;
+        font-size: 1em;
+        border: none;
+        box-shadow: 0 2px 0 #003618;
```
Comment (richardTowers) [inline lib/saml-form.ts]: Can we stick `cursor: pointer` in this too? There's some controversy over that:

https://ux.stackexchange.com/questions/3788/default-cursor-on-mouse-over-of-a-button-is-not-a-hand-pointer

But gov.uk elements puts the pointer on it (amongst some other things).

GOV.UK elements also has a few other bits and bobs for .button - I think:

```
.button {
  background-color: #00823b;
  position: relative;
  display: -moz-inline-stack;
  display: inline-block;
  padding: .526315em .789473em .263157em;
  border: none;
  -webkit-border-radius: 0;
  -moz-border-radius: 0;
  border-radius: 0;
  outline: 1px solid transparent;
  outline-offset: -1px;
  -webkit-appearance: none;
  -webkit-box-shadow: 0 2px 0 #003618;
  -moz-box-shadow: 0 2px 0 #003618;
  box-shadow: 0 2px 0 #003618;
  font-size: 1em;
  line-height: 1.25;
  text-decoration: none;
  -webkit-font-smoothing: antialiased;
  cursor: pointer;
  color: #fff;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
  vertical-align: top; }
  .button:visited {
    background-color: #00823b; }
  .button:hover, .button:focus {
    background-color: #00692f; }
  .button:active {
    top: 2px;
    -webkit-box-shadow: 0 0 0 #00823b;
    -moz-box-shadow: 0 0 0 #00823b;
    box-shadow: 0 0 0 #00823b; }
  .button.disabled, .button[disabled="disabled"], .button[disabled] {
    zoom: 1;
    filter: alpha(opacity=50);
    opacity: 0.5; }
    .button.disabled:hover, .button[disabled="disabled"]:hover, .button[disabled]:hover {
      cursor: default;
      background-color: #00823b; }
    .button.disabled:active, .button[disabled="disabled"]:active, .button[disabled]:active {
      top: 0;
      -webkit-box-shadow: 0 2px 0 #003618;
      -moz-box-shadow: 0 2px 0 #003618;
      box-shadow: 0 2px 0 #003618; }
  .button:link, .button:link:focus, .button:hover, .button:focus, .button:visited {
    color: #fff; }
  .button:before {
    content: "";
    height: 110%;
    width: 100%;
    display: block;
    background: transparent;
    position: absolute;
    top: 0;
    left: 0; }
  .button:active:before {
    top: -10%;
    height: 120%; }
  @media (max-width: 640px) {
    .button {
      width: 100%;
      text-align: center; } }

.button::-moz-focus-inner {
  border: 0;
  padding: 0; }

.button:focus {
  outline: 3px solid #ffbf47; }

.button[disabled="disabled"] {
  background: #00823b; }

.button[disabled="disabled"]:focus {
  outline: none; }
```

... not all of which is necessary, but we could maybe get a slightly better look and feel.

**Comment by RachaelBooth** [comment]
screen shot 2017-09-26 at 09 43 50

**Comment by RachaelBooth** [comment]
That screenshot doesn't actually show much obvious change, but there is now a slightly different green on hover (matching what @richardTowers posted for GOV.UK), the cursor: pointer has been added on the button (controversial!), and a small amount of padding has been added to the body.

Padding is not much because I couldn't get it to behave sensibly on small screens otherwise.

We don't need most of the GOV.UK button styling, I think, because we know the surrounding so we only have to catch the single case we're in.

File: lib/saml-form.ts [hunk 2]
```diff
@@ -6,17 +6,40 @@ import * as escape from 'escape-html'
...
+        font-size: 1em;
+        line-height: 1.25;
+        border: none;
+        box-shadow: 0 2px 0 #003618;
+        cursor: pointer;
+      }
+      .passport-verify-button:hover, .passport-verify-button:focus {
+        background-color: #00692f;
+      }
+    </style>
```
Comment (RachaelBooth) [inline lib/saml-form.ts:43]: This is starting to get quite long...

Comment (richardTowers) [reply lib/saml-form.ts:43]: Still looks ok to me, and the form looks really nice now. Good work 👍

**Pull request merged**